src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
Sorry for being unclear.  I am saying why return a Boolean or nullable type at all, since we don't need it.  The calling code does this:
```
FirmwareType fwType = FirmwareType.Unknown
if (Native.GetFirmwareType(ref fwType)) { rv.firmwareType = fwType; }
```

Since we are creating a wrapper why not have it just return the type:
```
rv.firmwareType = GetFirmwareType();
```

```
private static FirmwareType GetFirmwareType()
{
    try
    {
        FirmwareType fwType = FirmwareType.Unknown;
        if (Native.GetFirmwareType(ref fwType))
        {
            return fwType;
        }
    }
    catch (Exception) {  // Fail silently... }

    return FirmwareType.Unknown;
}
```